labs/lab-03/tasks/goto-algs/binary_search/support/Makefile

```
@@ -1,8 +1,8 @@
 CC = gcc
-CFLAGS = -Wall -g
+CFLAGS = -Wall
 LDFLAGS =
 
-TARGET_EXEC = binary-search
+TARGET_EXEC = main binary_search
```

```suggestion
TARGET_EXEC = main
```
The only executable is going to be `main`.